balancer-js/src/lib/utils/poolHelper.ts

```diff
@@ -62,20 +102,24 @@ export const parsePoolInfo = (pool: Pool, wrappedNativeAsset?: string) => {
 
   const parsedAmp = pool.amp
     ? parseFixed(pool.amp, AMP_PRECISION).toString() // Solidity maths uses precison method for amp that must be replicated
-    : undefined;
+    : defaultOne;
```

Comment:
It should be 1e3 I think, which is amp 1

Reply:
This is really just to avoid having to check for undefined whenever amp is being used. In reality AMP should always be defined if its needed and a default will be incorrect whatever value we use. Overall this can be improved by better typing for different pool types but don't want to add that in to the mix right now.